### PR TITLE
"fix" CmakeFile and add a build.jl

### PR DESCRIPTION
**Pull request by fieker**
which is not perfect (I don;t know what the conventions are)
but it illustrates what to do

### CODE REVIEW

File: build.jl [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#cmake -DJulia_EXECUTABLE=/home/fieker/Downloads/julia-1.5.3/bin/julia -DSingular_PREFIX=/home/fieker/.julia/dev/Singular/deps/usr/ -DCMAKE_INSTALL_PREFIX=/tmp -DJlCxx_DIR=/home/fieker/.julia/artifacts/860a8b2216bd059600ed7c44cdaa3bb81b23ff1c/lib/cmake/JlCxx -DCMAKE_BUILD_TYPE=Release -S libsingular-julia -B /tmp/build
```
Comment (thofma) [inline build.jl:1]:
```suggestion
#cmake -DJulia_EXECUTABLE=/home/bla/Downloads/julia-1.5.3/bin/julia -DSingular_PREFIX=/home/bla/.julia/dev/Singular/deps/usr/ -DCMAKE_INSTALL_PREFIX=/tmp -DJlCxx_DIR=/home/bla/.julia/artifacts/860a8b2216bd059600ed7c44cdaa3bb81b23ff1c/lib/cmake/JlCxx -DCMAKE_BUILD_TYPE=Release -S libsingular-julia -B /tmp/build
```

**Comment by thofma** [comment]
@fieker Should we build new binaries with the current version?

**Comment by fieker** [comment]
On Mon, Dec 07, 2020 at 02:50:59AM -0800, thofma wrote:
> @fieker Should we build new binaries with the current version?

Wait. I am playing with the total project.

File: build.jl [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#cmake -DJulia_EXECUTABLE=/home/bla/Downloads/julia-1.5.3/bin/julia -DSingular_PREFIX=/home/bla/.julia/dev/Singular/deps/usr/ -DCMAKE_INSTALL_PREFIX=/tmp -DJlCxx_DIR=/home/bla/.julia/artifacts/860a8b2216bd059600ed7c44cdaa3bb81b23ff1c/lib/cmake/JlCxx -DCMAKE_BUILD_TYPE=Release -S libsingular-julia -B /tmp/build
+
+import Singular, CxxWrap
+
+Julia_EXECUTABLE = joinpath(Sys.BINDIR, Base.julia_exename())
+Singular_PREFIX = joinpath(dirname(pathof(Singular)), "..", "deps", "usr")
```
Comment (benlorenz) [inline build.jl:6]: There should be no `deps` folder anymore, something like `Singular_jll.artifact_dir` would be better.

**Comment by fieker** [comment]
On Mon, Dec 07, 2020 at 03:00:36AM -0800, Benjamin Lorenz wrote:
> @benlorenz commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,13 @@
> +#cmake -DJulia_EXECUTABLE=/home/bla/Downloads/julia-1.5.3/bin/julia -DSingular_PREFIX=/home/bla/.julia/dev/Singular/deps/usr/ -DCMAKE_INSTALL_PREFIX=/tmp -DJlCxx_DIR=/home/bla/.julia/artifacts/860a8b2216bd059600ed7c44cdaa3bb81b23ff1c/lib/cmake/JlCxx -DCMAKE_BUILD_TYPE=Release -S libsingular-julia -B /tmp/build
> +
> +import Singular, CxxWrap
> +
> +Julia_EXECUTABLE = joinpath(Sys.BINDIR, Base.julia_exename())
> +Singular_PREFIX = joinpath(dirname(pathof(Singular)), "..", "deps", "usr")
> 
> There should be no `deps` folder anymore, something like `Singular_jll.artifact_dir` would be better.
> 
I agree - but as of now, I do not have Singular_jll....

**Pull request merged**